### DISCUSSION TITLE
adding dynamic or random values to header

### DISCUSSION THREAD

**Discussion by 0xAwali** [projectdiscovery/nuclei#4669] [Q&A]
it's possible to configure nuclei to use dynamic or random values to all templates e.g. I faced domain block every request unless using X-Forwarded-For: {{RANDOM-IP}} with every single template so can I configure nuclei to use X-Forwarded-For: {{RANDOM-IP}} in every single template e.g. configuration file

```
# Headers to include with each request.
header:
  - 'X-Forwarded-For: {{RANDOM-IP}}'
```

or using 

```
nuclei -H 'X-Forwarded-For: {{RANDOM-IP}}'
```

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
